zend_abstract_interface/config/php5/config_runtime.c

```diff
@@ -56,10 +123,8 @@ zval *zai_config_get_value(zai_config_id id) {
         TSRMLS_FETCH();
         return &EG(uninitialized_zval);
     }
-    if (runtime_config[id] == NULL) {
-        assert(false && "runtime config is not yet initialized");
```

Comment:
Is there a particular reason for dropping this assertion? If *runtime* config is accessed outside of a request stage that sounds like a violation of the config model